src/modules/drawing/DrawingModule.vue

```diff
@@ -243,7 +243,10 @@ export default {
             'setDrawingFeatures',
             'setKmlLayerAddToMap',
         ]),
-        async initDrawingOverlay() {
+        async showDrawingOverlay() {
```

Comment:
@pakb I renamed this function back to its previous name as I think it make more sense, we have a `showDrawingOverlay` and a `hideDrawingOverlay`.